src/App.css

```diff
@@ -19,16 +19,85 @@ body {
   padding: 0;
 }
 
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding: 3rem;
+}
+
+header h1 {
+  word-spacing: 0.5rem;
+  letter-spacing: 0.3rem;
+}
+
+header ul {
+  list-style-type: none;
+  display: flex;
```

Comment:
- [ ] You have used `display:flex` 6 times, which falls under the category of `repeating yourself`. I would suggest you create a class with a name similar to `d-flex` and give the `display:flex` and then use it at all places that you need it